nvchecker_source/apt.py

```diff
@@ -0,0 +1,60 @@
+# MIT licensed
+# Copyright (c) 2020 Felix Yan <felixonmars@archlinux.org>, et al.
+
+from nvchecker.api import session, GetVersionError
+
+APT_RELEASE_URL = "%s/dists/%s/Release"
+APT_PACKAGES_PATH = "%s/binary-%s/Packages%s"
+APT_PACKAGES_URL = "%s/dists/%s/%s"
+APT_PACKAGES_SUFFIX_PREFER = (".xz", ".gz", "")
+
+async def get_url(url):
+  res = await session.get(url)
+  data = res.body
+
+  if url.endswith(".xz"):
+    import lzma
+    data = lzma.decompress(data)
+  elif url.endswith(".gz"):
+    import gzip
+    data = gzip.decompress(data)
+
+  return data.decode('utf-8')
+
+async def get_version(name, conf, *, cache, **kwargs):
+  source_pkg = conf.get('source_pkg')
+  pkg = conf.get('pkg')
+  mirror = conf.get('mirror', "http://deb.debian.org/debian/")
+  suite = conf.get('suite', 'sid')
+  repo = conf.get('repo', 'main')
+  arch = conf.get('arch', 'amd64')
```

Comment:
Are these defaults special in some way? If not, I prefer them to be explicitly set.

Reply:
At least the defaults for repo and arch are generally okay IMHO. Perhaps one should set suite explicitly since it's so different for every distro.

Reply:
Updated to make `mirror` and `suite` mandatory.